WordPress/Classes/ViewRelated/Reader/ReaderDiscoverHeaderView.swift

```diff
@@ -22,10 +22,11 @@ final class ReaderDiscoverHeaderView: UIView, UITextViewDelegate {
         scrollView.clipsToBounds = false
         channelsStackView.pinEdges()
         scrollView.heightAnchor.constraint(equalTo: channelsStackView.heightAnchor).isActive = true
+        scrollView.contentInset = UIEdgeInsets(.leading, -10) // Align the "channels"
```

Comment:
I'm not sure how important it is to add these insets. Do the channels still align when increasing/decreasing the dynamic size?

Reply:
The margins in the channels don't scale with the dynamic type (for now), so it works OK. I actually totally forgot about the dynamic type, and I fixed it now and made a couple of more changes:

- Add support for Dynamic Type in Reader stream headers
- Fix an issue with background for channels in dark mode
- Fix an issue with Reader menu not using the large type

I also updated `scrollView.contentInset = UIEdgeInsets(.leading, -10)` to only work in non-compact mode. On iPhone, I think it looks better without this offset.

There is a known issue with Dynamic Type as the table view doesn't reload the header well when the dynamic size changes, but I think I'll just ignore it for now. I'm not sure how to best fix it, and it's a minor issues. Folks don't change dynamic type "dynamically".